GliaWidgets/Sources/Theme/Alert/MessageAlertConfiguration.swift

```diff
@@ -3,22 +3,25 @@ public struct MessageAlertConfiguration {
     /// Title of the alert.
     public var title: String?
 
-    /// Message of the alert. Include `{message}` template parameter in the string to display Glia's error message.
+    /// Message of the alert. Include `{message}` template parameter 
+    /// in the string to display Glia's error message.
     public var message: String?
 
-    /// Controls the appearance of the X button in the alert. Defaults to true. If this value is true, an X button
-    /// shows up on the alert so that the visitor can dismiss it, and the background gets dimmed. If the value is false,
-    /// no X button shows up, and the background doesn't get dimmed.
+    /// Controls the appearance of the X button in the alert. Defaults to true. 
+    /// If this value is true, an X button shows up on the alert so that the visitor
+    /// can dismiss it, and the background gets dimmed. If the value is false, no X button
+    /// shows up, and the background doesn't get dimmed.
```

Comment:
would it make sense to add `///` here also? or do we use additional `///` for initializers only?

Reply:
Good idea, makes it easier to read. Fixed [14839cf](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/14839cfb8b707e3a12197ff4bd96b79b48073da5)